src/CFamily.UnitTests/CFamily.UnitTests.csproj

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.4.0" />
     <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
-    <PackageReference Include="Moq" Version="4.14.5" />
+    <PackageReference Include="Moq" Version="4.18.2" />
```

Comment:
Upgraded to a version used by VS to avoid conflict. The impact on our tests turned out to be minimal so I thought it was worth upgrading the package overall, to be up to date.